u-root.go

```diff
@@ -76,7 +77,8 @@ func init() {
 
 	noCommands = flag.Bool("nocmd", false, "Build no Go commands; initramfs only")
 
-	flag.Var(&extraFiles, "files", "Additional files, directories, and binaries (with their ldd dependencies) to add to archive. Can be speficified multiple times.")
+	flag.Var(&extraFilesLdd, "lddfiles", "Additional files, directories, and binaries (WITH their ldd dependencies) to add to archive. Can be speficified multiple times.")
+	flag.Var(&extraFilesNoLdd, "files", "Additional files, directories, and binaries (WITHOUT their ldd dependencies) to add to archive. Can be speficified multiple times.")
```

Comment:
This is a breaking change  for every single script in existence -- and that's no small number. You'll need to not change the semantics of -files.

Reply:
I initially did this in response to @orangecms's [comment](https://github.com/u-root/u-root/issues/1920#issuecomment-773637687) about differentiating flags, though breaking existing scripts is a valid consideration I had not thought through thoroughly.